package.json

```diff
@@ -51,7 +51,7 @@
     "@webcomponents/shadycss": "latest",
     "@webcomponents/webcomponentsjs": "latest",
     "d2l-intl": "^2.0.0",
-    "intl-messageformat": "^3.0.0",
+    "intl-messageformat": "3.0.0",
```

Comment:
This package has a breaking change where they removed the `main.js` file. So locking at the most recent version that works.

Reply:
Can you try `lib/index.js` instead and see if that works? Looks like they converted to typescript or something.

Reply:
Yeah